synpress.json

```diff
@@ -0,0 +1,23 @@
+{
+  "baseUrl": "http://localhost:3000",
+  "userAgent": "synpress",
+  "retries": { "runMode": 0, "openMode": 0 },
+  "integrationFolder": "tests/e2e/specs",
+  "screenshotsFolder": "tests/e2e/screenshots",
+  "videosFolder": "tests/e2e/videos",
+  "fixturesFolder": "tests/e2e/fixtures",
+  "chromeWebSecurity": true,
+  "viewportWidth": 1366,
+  "viewportHeight": 768,
+  "component": {
+    "componentFolder": ".",
+    "testFiles": "**/*spec.{js,jsx,ts,tsx}"
+  },
+  "env": {
+    "coverage": false
+  },
+  "defaultCommandTimeout": 30000,
+  "pageLoadTimeout": 30000,
+  "requestTimeout": 30000,
+  "supportFile": "tests/e2e/support/index.js"
+}
```

Comment:
can set video + screenshots to `false`? dont think we need it, the devs can enable it if they need to debug (or use cy open instead)

Reply:
ok